api/foxpass_create_users.py

```diff
@@ -30,12 +38,18 @@ def main():
             try:
                 email = row[0]
                 username = row[1]
+                password = row[2]
             except IndexError:
                 continue
             data = {'email': email, 'username': username}
             r = requests.post(URL + ENDPOINT, headers=header, data=json.dumps(data))
-            print '{}: {}'.format(email, r.json())
+            if r.json()['status'] == 'ok' and password not "''":
```

Comment:
shouldn't have to check against `''`, the csv parser should be taking care of that. Maybe it only recognizes double quotes? Can you check?

Reply:
Checked, and updated.